lib/ui/pages/score/widgets/calculation_warning_widget.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_app/src/r.dart';
+
+class ScoreCalculationWarning extends StatelessWidget {
+  const ScoreCalculationWarning({super.key});
+
+  @override
+  Widget build(BuildContext context) => Padding(
+        padding: const EdgeInsets.all(5),
+        child: Row(
+          children: [
+            Expanded(
+              child: Text(
+                R.current.scoreCalculationWarring,
```

Comment:
```suggestion
                R.current.scoreCalculationWarning,
```

Reply:
addressed in [7b1914e](https://github.com/NEO-TAT/tat_flutter/pull/157/commits/7b1914ed2b325a43d827f4ec2b4cc7f0a3d47bd7).